src/business-logic/services/interfaces/data-change-service.ts

```diff
@@ -0,0 +1 @@
+export interface DataChangeService {}
```

Comment:
Not sure if it should be `changed` or not.

Reply:
I debated this as well.. I could change both the ignest service and this to be `changed` instead?

Reply:
Not sure what makes most sense, so we can change it or leave it as is